code/modules/research/designs.dm

```diff
@@ -312,7 +312,7 @@
 	materials = list(MATERIAL_STEEL = 6000, MATERIAL_GLASS = 1000) //expensive, but no need for miners.
 	build_path = /obj/item/weapon/tool/pickaxe/drill
 	sort_string = "KAAAB"
-*/
```

Comment:
I'm not quite clear what you're doing here, or why. Can you explain it farther?

Reply:
The mining cutter, specifically the datum and ID is used 3 times in the file, I'm not sure for what reason, but that seems heavily unusual to me as it's used further on (under the weapon designs) and seemed unneeded.